src/macaron/dependency_analyzer/dependency_resolver.py

```diff
@@ -46,20 +52,19 @@ def __init__(self, resources_path: str, file_name: str, debug_path: str, tool_ve
             The path to the resources directory.
         file_name : str
             The name of dependency output file.
-        debug_path : str
-            The file path where all the dependencies will be stored for debugging.
+        tool_name: str
+            The name of the dependency analyzer.
         tool_version : str
             The version of the dependency analyzer.
+        repo_path: str
```

Comment:
This attribute is only needed for the `CycloneDxMaven` class for now. Do you think it is a good idea to move it there? :thinking:

Reply:
I think it's a general enough attribute that would be useful for other dependency resolvers down the road?